content/docs/components/code/theming.mdx

```diff
@@ -5,4 +5,156 @@ title: Code
 package: '@chakra-ui/layout'
 ---
 
-// TODO
+The `Code` component is a single part component. All of the styling is applied
+directly to the `code` element.
+
+> To learn more about styling single part components, visit the
+> [Component Style](/docs/styled-system/component-style#styling-single-part-components)
+> page.
+
+## Theming properties
+
+The properties that affect the theming of the `Code` component are:
+
+- `variant`: The visual variant of the code. Defaults to `subtle`.
+- `colorScheme`: The color scheme of the code. Defaults to `gray`.
+
+## Theming utilities
+
+- `defineStyle`: a function used to create style objects.
+- `defineStyleConfig`: a function used to define the style configuration for a
+  single part component.
+
+```jsx live=false
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+```
+
+## Customizing the default theme
+
+```jsx live=false
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+
+const outline = defineStyle({
+  border: '2px dashed', // change the appearance of the border
+  borderRadius: 0, // remove the border radius
+  fontSize:'md', // change the font size
+  fontWeight: 'semibold', // change the font weight
+})
+
+export const codeTheme = defineStyleConfig({
+  variants: { outline },
+})
+```
+
+After customizing the code theme, we can import it in our theme file and add
+it in the `components` property:
+
+```jsx live=false
+import { extendTheme } from '@chakra-ui/react'
+import { codeTheme } from './components/code.ts'
+
+export const theme = extendTheme({
+  components: { Code: codeTheme},
+})
+```
+
+> This is a crucial step to make sure that any changes that we make to the
+> Code theme are applied.
+
+## Adding a custom variant
+
+Let's assume we want to include a custom branded variant. Here's how we can do
+that:
+
+```jsx live=false
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+
+const brandPrimary = defineStyle({
+  background: 'orange.500',
+  color: 'white',
+  fontFamily: 'serif',
```

Comment:
A `serif` font wouldn't make sense for the `Code` component. It should always be `mono`.

Reply:
by default it's mono so i though to write something else, ill switch to mono then